### PR TITLE
new feature for login kaizen manager

### LINKED ISSUE #60: user authorized mechanism
In order to support the KAIZEN manager, KAIZEN CLI should have the user authorized mechanism to log in and log out KAIZEN manager.

### PR DESCRIPTION
**Pull request by ahlumin**
To resolve #60 issue

#### command 
`$ kaizen login`

#### usage
In order to integrate with KAIZEN manager, the user should request the authorization via login account in KAIZEN manager system.

Execute this command and input your email and password
image

* * * *
#### command 
`$ kaizen logout`

#### usage
To log out KAIZEN manager

image

### CODE REVIEW

File: src/components/Login/index.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
+const path = require('path');
+const fsx = require('fs-extra');
+const prompt = require('prompt');
+const Log = require('../../lib/Log');
+const Spinner = require('../../lib/Spinner');
+const axios = require('axios');
+const LOGIN_API = 'https://api.portal.network/user/v1/signIn';
```
Comment (PhyrexTsai) [inline src/components/Login/index.js]: We can have a file to store all API

File: .kaizenrc [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "name": ""
```
Comment (PhyrexTsai) [inline .kaizenrc]: Seems we need these parameters
```
accessToken,
refreshToken,
idToken,
expiresIn,
name
```

File: build/components/Login/index.js [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+      if (error) {
+        reject(error);
+      } else {
+        resolve(result);
+      }
+    });
+  });
+}
+
+function postLogin(email, password) {
```
Comment (PhyrexTsai) [inline build/components/Login/index.js]: Should we move to a file or library that support API?

**Comment by PhyrexTsai** [comment]
@ahlumin I leave some comment for you, please take a look, thank you

**Comment by ahlumin** [comment]
I refactored the codebase about API operating, please take a reference on [commit](https://github.com/PortalNetwork/kaizen-cli/pull/63/commits/db6763e04353a8ee6d8ab1a9a41eab7f610d4aed)

**Comment by PhyrexTsai** [comment]
@ahlumin This code looks nice, can you please add the documentation about login and logout.

Here's an example for the command how it looks like, you can check out this repository.
https://github.com/faressoft/terminalizer

**Pull request merged**